addons/l10n_ar/views/report_invoice.xml

```diff
@@ -272,6 +272,29 @@
             <attribute name="t-call">l10n_ar.document_tax_totals</attribute>
         </t>
 
+        <div id="qrcode" position="after">
+            <!-- RG 5614/2024: Show ARCA VAT and Other National Internal Taxes -->
+            <t t-set="l10n_ar_custom_tax_summary" t-value="o._l10n_ar_get_invoice_custom_tax_summary_for_report()"/>
+            <div class="l10n_ar_tax_details col-sm-7 col-md-6" t-if="l10n_ar_custom_tax_summary">
```

Comment:
table widht is small. please change the class to this way in order to make it larger
 
![image](https://github.com/user-attachments/assets/fbb6301a-9ce4-4f86-ac40-b5e5aa8eb8be)
```suggestion
            <div class="l10n_ar_tax_details" t-if="l10n_ar_custom_tax_summary">
```

Reply:
@smetl forget to ping you in the last comment, Already reviewed functionaly with @vbe-odoo